### DISCUSSION TITLE
Can PULID be enhanced by the new Flux model?

### DISCUSSION THREAD

**Discussion by zimbora95** [ToTheBeginning/PuLID#57] [Ideas]
I’m curious about the capabilities of the new Flux model compared to the stable diffusion models. Specifically, will the Flux model improve the overall quality of generated images and better adhere to prompts for PulID?

**Comment by nullarch** [comment]
Possible. But this is not managed project last commit is 4month ago. I think instantX team will train flux with faceID.

**Comment by ToTheBeginning** [comment]
PuLID for FLUX is released now, please enjoy and have fun with it!

**Reply by zimbora95** [reply]
Thank you so much! 🥇
